tests/vectors/inputs/jwkEd25519Private1.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Comment for overall test input file naming:

Be consistent with naming with the other secp2561k files. ie. Have the number (1, 2 etc) in the middle as opposed to the end. I did that so that public and private key files are next to each other making it obviously that the files come in pairs.

Reply:
Done here: https://github.com/decentralized-identity/ion-sdk/pull/20/commits/51a810c6d4ad4031f308b9aaaccd67de17f865f5

For context, I put it at the end because I found that the names `Ed255191`, `Ed255192`, `Ed255193` looked a bit weird (hard to distinguish the name of the algo vs the 1 2 3)

Reply:
@gjgd, valid point. Maybe flipping the name to `ed255191jwk1private` would be the best compromise. But won't hold the PR on this, singing off.